src/Toolchain.ml

```diff
@@ -254,12 +282,27 @@ let select () =
   let workspaceFolders = Vscode.Workspace.workspaceFolders () in
   sandboxCandidates ~workspaceFolders >>= fun candidates ->
   let open Promise.Option.O in
-  selectPackageManager candidates >>= fun { status; packageManager } ->
-  match status with
-  | Error s ->
-    message `Warn "This toolchain is invalid. Error: %s" s;
-    Promise.return None
-  | Ok () -> Promise.Option.return packageManager
+  selectPackageManager candidates >>= function
+  | { status = Ok (); packageManager = Custom _ } ->
+    let validateInput input =
+      if Js.String.includes "$prog" input && Js.String.includes "$args" input
```

Comment:
Actually, vscode already has a standard syntax for this stuff: `${prog}`, `${args}`. I suggest that we just reuse it.

I'm trying to find an API in vscode that would allow us to expand stuff like this, but without any luck so far.